### PR TITLE
#604 - RPM list and install

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #604 
Added test for `yum list` and `install`

### CODE REVIEW

File: src/test/java/com/artipie/rpm/RpmITCase.java [hunk 1]
```diff
@@ -91,31 +115,70 @@ void addsRpmAndCreatesRepodata() throws Exception {
...
             "Repository xml indexes are created",
-            new FileStorage(this.tmp).list(new Key.From("my-rpm/repodata")).join().size(),
+            new FileStorage(this.tmp).list(new Key.From(RpmITCase.REPO, "repodata")).join().size(),
             new IsEqual<>(4)
         );
         con.disconnect();
     }
 
+    @Test
+    void listAndInstallsRpmPackage() throws Exception {
```
Comment (olegmoz) [inline src/test/java/com/artipie/rpm/RpmITCase.java:125]: @olenagerasimova is there reason to test install and list operations in single test? I'd suggest to create separate test for these operations

**Review by olegmoz — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see one comment

**Comment by olenagerasimova** [comment]
@olegmoz thanks, I've split the test

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks, merging

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#666` is not assigned, can't get performer

**Pull request merged**